src/components/DeFi/components/Steps.tsx

```diff
@@ -20,15 +20,19 @@ export type DefiStepProps = { [key in DefiStep]?: StepConfig }
 type StepsProps = {
   steps: DefiStepProps
   initialStep?: DefiStep
+  keepStatusInTheStepper?: boolean
 } & StackProps
 
 export const Steps: React.FC<StepsProps> = ({
   steps,
   children,
   initialStep = DefiStep.Info,
+  keepStatusInTheStepper,
   ...rest
 }) => {
-  const otherSteps = Object.keys(steps).filter(e => e !== DefiStep.Status)
+  const otherSteps = Object.keys(steps).filter(e =>
+    keepStatusInTheStepper ? true : e !== DefiStep.Status,
+  )
```

Comment:
We could make this a one-liner.

```suggestion
  const otherSteps = Object.keys(steps).filter(e => keepStatusInTheStepper ?? e !== DefiStep.Status)
```